homeassistant/components/roomba/vacuum.py

```diff
@@ -135,6 +78,20 @@ def __init__(self, name, roomba):
         self.vacuum = roomba
         self.vacuum_state = None
 
+    @property
+    def unique_id(self):
+        """Return the uniqueid of the vacuum cleaner."""
```

Comment:
Is there a serial number can be used here instead?
https://developers.home-assistant.io/docs/entity_registry_index/

Reply:
now the unique_id is mac address